### PR TITLE
feat: error component

### PR DESCRIPTION
**Pull request by cindy-sf**

### CODE REVIEW

File: src/components/Error/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import ErrorImage from '@assets/images/error/error_image.png'
+
+import Button from '@components/Button'
+import Image from '@components/Image'
+import Layout from '@components/Layout'
+import Text from '@components/Text'
+
+const Error = ({ navigation }: { navigation: any }) => (
+  <Layout>
+    <View style={{ flex: 2 }}>
```
Comment (j0nathansf) [inline src/components/Error/index.tsx]: Why didn't you use styled components?

File: src/components/Error/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <Text font="POPPINS_SEMI_BOLD" size="HEADLINE_1" maxWidth={320}>
+        Navré, une erreur est survenue...
+      </Text>
+      <View style={{ flex: 0.4 }} />
```
Comment (j0nathansf) [inline src/components/Error/index.tsx]: Same

File: src/components/Error/index.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      <Image src={ErrorImage} height={285} width={270} resizeMode="cover" />
+      <View style={{ flex: 0.4 }} />
+    </View>
+    <View style={{ alignItems: 'center', flex: 0.5 }}>
```
Comment (j0nathansf) [inline src/components/Error/index.tsx]: Here too....

File: src/components/Loader/index.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <Layout>
+      <View
+        style={{
+          flex: 1,
+          justifyContent: 'center',
+          alignItems: 'center',
+          width: windowWidth,
+          marginBottom: windowHeight / 5.5,
+        }}
+      >
```
Comment (j0nathansf) [inline src/components/Loader/index.tsx]: I think we'll have to review this

**Review by j0nathansf — CHANGES_REQUESTED** [review]
Retouch your style elements with styled components and it's OK

File: src/components/Loader/index.tsx (hunk 4 above)
Comment (cindy-sf) [reply src/components/Loader/index.tsx]: Yes 👍

**Review by j0nathansf — APPROVED** [review]
Parfait

**Pull request merged**